modules/nf-neuro/image/burnvoxels/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: "image_burnvoxels"
+description: Burn voxels from masks into an anatomical image.
```

Comment:
This might be a silly comment, but would you mind explaining the module behavior? I am unsure what burning voxels means, and if someone wants to use it in the future, it might be a good idea to add some details in the description.